ui-modules/utils/quick-launch/quick-launch.js

```diff
@@ -38,7 +38,7 @@ export function quickLaunchDirective() {
         scope: {
             app: '=',
             locations: '=',
-            args: '=?',
+            args: '=?', // default: { noEditButton: false, noComposerButton: false, noCreateLocationLink: false, location: null }
```

Comment:
Where this default comes from? It not enforce anywhere nor used.

Reply:
the values show what the default behaviour according to the code is